packages/cli/src/commands/test.ts

```diff
@@ -42,8 +42,16 @@ async function storeTest(tmpDir: string) {
 }
 
 export default class Test extends Command {
+  static args = [
+    {
+      name: 'path',
+      description: 'The path where the FastStore being tested is. Defaults to cwd.',
+    }
+  ]
```

Comment:
suggestion: What do you think about adding an example using `defaultHelp`?

Reply:
The version of `oclif/core` being used does not have the `defaultHelp` argument 😰